src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('#employeeList > li');
```

Comment:
```suggestion
const listItems = document.querySelectorAll('li');
```
selector can be simplier